### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (p)

### ISSUE THREAD

**Issue by geraldo128** [geraldo128/testMe#1]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI))
#### Target application
[Django and SMS App Development Case Studies | Caktus Group](https://www.caktusgroup.com/casestudies/)
#### Element path
.blog-card-wrapper.full:nth-child(2) > .card:nth-child(1) > .main-link > .card-info > .sample-text > p
#### Snippet
`<p>
              For its innovative healthcare solution, Force Therapeutics needed to drive scalability and guard against technical debt.
            </p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.7 (foreground color: #7b8786, background color: #ffffff, font size: 12.3pt (16.456px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 86.0.4240.193
This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
